src/features/MentorPage/MentorChips/MentorChips.tsx

```diff
@@ -0,0 +1,74 @@
+import { ChipProps } from '../../../components/Chip/types';
+import styled from 'styled-components';
+import Chip from '../../../components/Chip';
+import ShowMoreChips from './ShowMoreChips';
+import React from 'react';
+
+/**
+ * Logic to show skill chips. Array of skill names is
+ * mapped and passed to Chip one by on, where styles and layout
+ * will be added.
+ */
+
+const MentorSkillChips = ({ items }: { items: Array<ChipProps> }) => {
+  const [shouldShowAllSkills, setShowAllSkills] = React.useState(false);
+
+  const handleShowMoreSkillsChange = () =>
+    setShowAllSkills(!shouldShowAllSkills);
+
+  //change the first letter of every skill to uppercase, then alphabetize
+  const upperCaseSkills = items.map(item => {
+    const newItem: ChipProps = {
+      text: item.text.charAt(0).toUpperCase() + item.text.slice(1),
+    };
+    return newItem;
+  });
```

Comment:
This can also be shorthanded like this

```javascript
 const upperCaseSkills = items.map(item => ({
 ...item, 
 text: `${item.text.charAt(0).toUpperCase()}${item.text.slice(1)}`,
    })
  });
  ```
  
  But as I said in other comment, there is no need for this =P you can lowercase on sort